src/ethereum/shanghai/fork.py

```diff
@@ -411,7 +413,8 @@ def apply_body(
     transactions: Tuple[Union[LegacyTransaction, Bytes], ...],
     chain_id: U64,
     withdrawals: Tuple[Withdrawal, ...],
-) -> Tuple[Uint, Root, Root, Bloom, State, Root]:
+    inclusion_list_summary: InclusionListSummary,
+) -> Tuple[Uint, Root, Root, Bloom, State, Root, Root, Root]:
```

Comment:
I see two extra `Root` fields appended to the retval here, but no update to the `Returns` section of the comment below, nor to the `return` at the end of the function.